lib/jsdom/living/helpers/style-rules.js

```diff
@@ -77,14 +84,14 @@ function getCascadedPropertyValue(element, property) {
 
 // https://drafts.csswg.org/css-cascade-4/#specified-value
 function getSpecifiedValue(element, property) {
+  const { initial, inherited, coerce } = exports.propertiesWithResolvedValueImplemented[property];
   const cascade = getCascadedPropertyValue(element, property);
 
   if (cascade !== "") {
-    return cascade;
+    return coerce instanceof Function ? coerce(cascade) : cascade;
```

Comment:
Let's just assume it's the right type and get a loud error if not:

```suggestion
    return coerce ? coerce(cascade) : cascade;
```